SourceX/qol.cpp

```diff
@@ -155,39 +190,103 @@ void DrawXPBar(CelOutputBuffer out)
 	if (!sgOptions.Gameplay.bExperienceBar)
 		return;
 
-	int barWidth = 306;
-	int barHeight = 5;
-	int yPos = gnScreenHeight - 9;                 // y position of xp bar
-	int xPos = (gnScreenWidth - barWidth) / 2 + 5; // x position of xp bar
-	int dividerHeight = 3;
-	int numDividers = 10;
-	int barColor = 198;
-	int emptyBarColor = 0;
-	int frameColor = 196;
-	bool space = true; // add 1 pixel separator on top/bottom of the bar
-
-	PrintGameStr(out, xPos - 22, yPos + 6, "XP", COL_WHITE);
-	int charLevel = plr[myplr]._pLevel;
-	if (charLevel == MAXCHARLEVEL - 1)
+	const PlayerStruct& player = plr[myplr];

```

Comment:
Can't see anything in the coding style guide about this. Ultimately, as long as only a single variable per line is declared (which is generally regarded as a good base rule), it doesn't matter.

Reply:
Yes but it will cause differences when autoformatted (at least my VS2019 autoformats & to the right side and I'm assuming it's doing what devilutionx is telling it to

Reply:
I'm using the clang-tidy that came with the project (at least I think I am), but I don't think the rules in there are quite correct. It for example complains about variable names, since the rules state that all variable names have to be exclusively lower case.

Reply:
I guess we have to ask @AJenbo

Reply:
clang-format is for style.

clang-tidy is for refactoring and has not been applied or adjusted yet.

Reply:
Ah! Will investigate how to teach VS that...

Reply:
I copy/paste my code 😅

Reply:
https://devblogs.microsoft.com/cppblog/clangformat-support-in-visual-studio-2017-15-7-preview-1/